pilot/lib/pilot/manager.rb

```diff
@@ -13,7 +13,11 @@ class Manager
     def start(options, should_login: true)
       return if @config # to not login multiple times
       @config = options
-      login if should_login
+
+      # we will always start with App Store Connect API login if required and possible
+      # else fallback to 'should_login' param for 'apple_id' login
+      should_login_for_appstore_connect_api = Spaceship::ConnectAPI.token.nil? && (options[:api_key_path] || options[:api_key])
```

Comment:
I _think_ we might want to remove the `Spaceship::ConnectAPI.token.nil?` 🤔 There could be a chance that the account that is logged in with `Spaceship::ConnectAPI.token` is different than the team that `:api_key_path` or `:api_key` are specifying. This would ignore the other team that is trying to be used and keep the one that is already authenticated.

Thoughts?

Reply:
I agree!